test/3D_laplacian.jl

```diff
@@ -0,0 +1,36 @@
+using DiffEqOperators, DifferentialEquations, ProgressMeter
+function main()
```

Comment:
@xtalax note that this isn't needed anymore now that we are using `@safetestsets`

Reply:
noted, i'll change that around on my next commit